doc/man/scons.xml

```diff
@@ -6496,17 +6508,17 @@ echo Last build occurred  . &gt; $TARGET
 <title>Python Code Substitution</title>
 
 <para>
-Any Python code within curly braces
-(<emphasis role="bold">{}</emphasis>)
-and introduced by the variable prefix <literal>$</literal>
-will be evaluated using the Python <function>eval</function> statement,
-with the Python globals set to
-the current environment's set of &consvars;, and the result
-substituted in.
+If a substitutable expression using the notation
+<literal>${something}</literal> does not appear to match one of
+the other substitution patterns,
+it is evaluated as a Python expression.
+This uses Python's <function>eval</function> function,
+with the <parameter>globals</parameter> parameter set to
+the current environment's set of &consvars;,
+and the result substituted in.
 So in the following case:</para>
 
 <programlisting language="python">
-env['COND'] = 0
 env.Command('foo.out', 'foo.in',
     '''echo ${COND==1 and 'FOO' or 'BAR'} &gt; $TARGET''')
```

Comment:
does this blow up when there's no COND defined? (shouldn't.. but.)

Reply:
I'll check and add a note if it does. I was trying to fix the logical problem that example exactly set a value and then wording said "depending on the value".

Reply:
It doesn't error, the substitution returns an empty string.  The result is also not quite what I expect: if completely undefined: 
```
echo > foo.out
```